cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,104 @@
+. Click *Create* to create the new private space.
++
+The *Upgrade to CloudHub 2.0* review page shows the progress of the VPC upgrade process, which takes approximately fifteen to thirty minutes.
+
+As soon as the upgrade process starts, the upgraded VPC is locked for any further updates. 
+
+You can cancel the upgrade process to revert to your VPC at any time by clicking *Cancel Upgrade*. In that case, the newly created private space is deleted, and your VPC is unlocked.
+
+Once you have successfully provisioned the private space, the *Cancel Upgrade* button is no longer visible. However, you can still roll back the upgrade by deleting the private space.
+
+
+== Next Steps after VPC Upgrade
+
+After you have completed the upgrade, the private space is fully provisioned and you can start deploying applications into it.
+
+You can start migrating your applications individually by creating new versions in the CloudHub 2.0 private space and then deleting the corresponding application in the CloudHub VPC. Once you’ve successfully migrated all your applications, make sure to delete your old VPC and any attached Dedicated Load Balancers on CloudHub.
+
+During your migration, you can continue to deploy and manage applications in your VPC to maintain business continuity. However, after the upgrade, your CloudHub Dedicated VPC is locked for any infrastructure changes to prevent configuration drift. This includes adjustments to environment or business group associations, network connections, and internal DNS. Make all changes in the newly created private space instead. Any configuration changes carried over to the CloudHub 2.0 private space are applied to the CloudHub VPC during your upgrade. 
+
+If your CloudHub VPC doesn't have any network connections and you create new ones in your new CloudHub 2.0 private space after the upgrade, these new connections aren’t reflected back in your CloudHub VPC.
+
+The firewall rules configured in your VPC get copied to the private space during the upgrade. After that, you can make additional changes to the firewall rules in both the VPC and the private space. To avoid configuration drift, apply any firewall rule changes in both places throughout your migration.
+
+== Entitlements for VPCs, Private Spaces, and Network Connections (VPNs/TGW)
+
+Once you complet the VPC upgrade and bring over any existing TGW and VPN network connections, both your old and new infrastructure are counted separately towards your usage. Therefore, if your organization is already using up all of your purchased quota, your usage can exceed the allowed quota until you move all your applications to CloudHub 2.0 and decommissioned your old VPC. 
+
+Work with your account representative to plan for additional entitlements, and make sure you decommission your CloudHub infrastructure (VPCs and associated DLBs) within six months after the upgrade to avoid interruptions when trying to create or edit VPCs, private spaces, or network connections.
```

Comment:
```suggestion
Work with your account representative to plan for additional entitlements, and decommission your CloudHub infrastructure (VPCs and associated DLBs) within six months after the upgrade to avoid interruptions when trying to create or edit VPCs, private spaces, or network connections.
```